### ISSUE TITLE
Renaming a Workflow Run only works the second time

### ISSUE THREAD

**Issue by PythonSemicolon** [DDMAL/Rodan#975]
After double-clicking a Workflow Run in a Rodan project and changing the name, upon clicking "Save", a JS error is thrown several times in the console:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'each')
    at BehaviorTable._getFilterLabels (BehaviorTable.js:259:1)
    at BehaviorTable._getFilters (BehaviorTable.js:149:1)
    at BehaviorTable._injectFiltering (BehaviorTable.js:198:1)
    at BehaviorTable._handleCollectionEventSync (BehaviorTable.js:420:1)
    at triggerEvents (backbone.js:337:1)
    at triggerApi (backbone.js:322:1)
    at eventsApi (backbone.js:110:1)
    at Events.trigger (backbone.js:312:1)
    at options.success (backbone.js:1061:1)
    at options.success (ControllerServer.js:239:1)
```

Furthermore, nothing concrete happens. Clicking "Save" a second time after this throws more of the same error in the console, but opens the loading modal, which seems to never close (waited about a minute and nothing happened). However, after manually closing the modal and returning to the Workflow Runs page, the name is changed.

Note: deleting a Workflow run seems to work on the first try, but also throws the same JS error.

**Comment by jackyyzhang03** [comment]
Could this have been fixed by #980?

**Comment by PythonSemicolon** [comment]
> Based on the error, this appears to be the same issue as #977. Could this have been fixed by #980?

I will pull and check, it may very well be the case.

**Comment by malajvan** [comment]
Recently tested, seems to have been fixed as @jackyyzhang03 mentioned above.
